influxql/query_manager.go

```diff
@@ -222,10 +222,12 @@ func (qm *defaultQueryManager) Query(qid uint64) (*queryTask, bool) {
 	return query, ok
 }
 
-func (qm *defaultQueryManager) waitForQuery(qid uint64, timeout time.Duration, closing <-chan struct{}, monitorCh <-chan error) {
+func (qm *defaultQueryManager) waitForQuery(qid uint64, interrupt <-chan struct{}, timeout time.Duration, closing <-chan struct{}, monitorCh <-chan error) {
 	var timer <-chan time.Time
```

Comment:
`sync.Pool` does grow and shrink dynamically with usage, but I don't think that we should try to optimize that out before we know whether or not it's a problem.